test/Performance/setup_env_unix.sh

```diff
@@ -1,140 +1,184 @@
-    echo "OK"
-    echo "Installing wget..."
-    brew install wget >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing svn..."
-    brew install svn >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing openssl..."
-    brew install pkg-config >> env_setup.log 2>&1
-    brew install openssl >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing python3..."
-    brew install python3 >> env_setup.log 2>&1
-    echo "OK"
-    echo "Installing MSODBCSQL..."
-    brew tap microsoft/msodbcsql https://github.com/Microsoft/homebrew-msodbcsql >> env_setup.log 2>&1
-    brew update >> env_setup.log 2>&1
-    yes | ACCEPT_EULA=Y brew install msodbcsql >> env_setup.log 2>&1
-    echo "OK"
-    yes | brew install autoconf >> env_setup.log 2>&1
-    echo "Installing pyodbc..."
-    pip3 install pyodbc >> env_setup.log 2>&1
-    echo "OK"
+    printf "Installing homebrew..."
+    yes | /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" >> env_setup.log
+    printf "done\n"
+
+    printf "Installing wget..."
+    brew install wget >> env_setup.log
+    printf "done\n"
+
+    printf "Installing svn..."
+    brew install svn >> env_setup.log
+    printf "done\n"
+
+    printf "Installing openssl..."
+    brew install pkg-config >> env_setup.log
+    brew install openssl >> env_setup.log
+    printf "done\n"
+
+    printf "Installing python3..."
+    brew install python3 >> env_setup.log
+    printf "done\n"
+
+    printf "Installing MSODBCSQL..."
+    brew tap microsoft/msodbcsql https://github.com/Microsoft/homebrew-msodbcsql >> env_setup.log
+    brew update >> env_setup.log
+    yes | ACCEPT_EULA=Y brew install msodbcsql >> env_setup.log
+    printf "done\n"
+
+    yes | brew install autoconf >> env_setup.log
+    printf "Installing pyodbc..."
+    pip3 install pyodbc >> env_setup.log
+    printf "done\n"
 fi
-echo "Downloading PHP-$PHP_VERSION source tarball..."
+
+printf "Downloading PHP-$PHP_VERSION source tarball..."
+#   pipe non-error to log file
 wget http://ca1.php.net/get/php-$PHP_VERSION.tar.gz/from/this/mirror -O php-$PHP_VERSION.tar.gz >> env_setup.log 2>&1
-echo "OK"
-echo "Extracting PHP source tarball..."
+printf "done\n"
+
+printf "Extracting PHP source tarball..."
 rm -rf php-$PHP_VERSION
 tar -xf php-$PHP_VERSION.tar.gz
-echo "OK"
-cd php-$PHP_VERSION
-mkdir ext/sqlsrv ext/pdo_sqlsrv
-cp -r $DRIVER_SOURCE_PATH/sqlsrv/* $DRIVER_SOURCE_PATH/shared ext/sqlsrv
-cp -r $DRIVER_SOURCE_PATH/pdo_sqlsrv/* $DRIVER_SOURCE_PATH/shared ext/pdo_sqlsrv
+printf "done\n"
+
+phpDir=php-$PHP_VERSION
+cd $phpDir
+
+printf "Configuring PHP..."
 ./buildconf --force >> ../env_setup.log 2>&1
-CONFIG_OPTIONS="--enable-cli --enable-cgi --enable-pdo --enable-sqlsrv=shared --with-pdo_sqlsrv=shared --with-odbcver=0x0380 --with-zlib --enable-mbstring --prefix=/usr/local"
+CONFIG_OPTIONS="--enable-cli --enable-cgi --with-zlib --enable-mbstring --prefix=/usr/local"
```

Comment:
Has it been tested in all platforms? the loading behavior of PDO drivers changes depending on all platforms